scripts/python/upload-with-submit.py

```
@@ -0,0 +1,239 @@
+        }
+
+        headers = {
+            "Accept": "*/*",
+            "Origin": "https://gemsofindia.org",
+            "Referer": "https://gemsofindia.org/submit",
+            "User-Agent": "Mozilla/5.0",
+            "x-uploadthing-version": "7.7.4",
+        }
+
+        try:
+            r = requests.put(upload_url, files=files, headers=headers)
```

Ruff (0.14.4)

Probable use of `requests` call without timeout

(S113)